numpy/core/defchararray.py

```diff
@@ -318,6 +318,21 @@ def add(x1, x2):
         Output array of `bytes_` or `str_`, depending on input types
         of the same shape as `x1` and `x2`.
 
+    Examples
+    --------
+    >>> a = np.array(["a", "b", "c"])
+    >>> np.char.add("0", a)
+    array(['0a', '0b', '0c'], dtype='<U2')
+
```

Comment:
```suggestion
```
If there are no comments between the examples, probably should either remove the whitespace or add `>>>` to the blank lines so there are not separate code blocks.